src/MatrixClientPeg.ts

```diff
@@ -228,9 +242,9 @@ class _MatrixClientPeg {
             ],
             unstableClientRelationAggregation: true,
             identityServer: new IdentityAuthClient(),
+            cryptoCallbacks: {},
         };
 
-        opts.cryptoCallbacks = {};
         // These are always installed regardless of the labs flag so that
         // cross-signing features can toggle on without reloading and also be
```

Comment:
Can we just use window and define it in global.d.ts please

Reply:
Presumably this is supposed to be several lines lower.

Why aren't we using the DOM types?

Reply:
We are using the dom types. Global.d.ts lists extensions to the window interface, such as modernizr etc